src/main/java/org/vietabroader/model/VASpreadsheet.java

```diff
@@ -15,8 +17,75 @@
  */
 public class VASpreadsheet {
 
+
     private String spreadsheetId;
     private Spreadsheet spreadSheet;
+    private int fromRow;
+    private int toRow;
+    private HashMap<String,String> columnNameToChar;
+    private HashMap<String,List<Object>> cachedColumns;
+
+    /**
+     * Init the maps
+     */
+    public void init(){
+        columnNameToChar = new HashMap<String, String>();
+        cachedColumns = new HashMap<String,List<Object>>();
+    }
+
+    /**
+     * Set the column
+     * @param columnName the name of the column
+     * @param character the character of the column
+     */
+     public void setColumnChar(String columnName, String character){
+         columnNameToChar.put(columnName,character);
+     }
+
+    /**
+     * Set the row
+     * @param rowStart the first row
+     * @param rowEnd the last row
+     */
+     public void setRow(int rowStart, int rowEnd){
+         fromRow = rowStart;
+         toRow = rowEnd;
+     }
+
+    /**
+     * Read first key from the Item column for testing the refreshColumn method
+     * @param sheetName name of the sheet
+     * @param col name of the column
+     * @return the first key in the column
+     * @throws IOException
+     * @throws GeneralSecurityException
+     */
+    public String readFirstKey(String sheetName, String col) throws IOException, GeneralSecurityException{
```

Comment:
Don't put test code here. Only do that when there is no other way. In this case, create `List<Object> readCol(String col)` to read the whole col. This method is both useful for testing and for future use.